### DISCUSSION TITLE
App not opening and skipping tests but shown as successful.

### DISCUSSION THREAD

**Discussion by AndyInso** [leancodepl/patrol#1629] [General]
I thought I'd share this in case others run into it. I ran into the same issue on multiple apps so took a step back to basics.

Two demo app projects, identical setup and a simple failing test.

Patrol test ran and correctly showed the failing test on one of these.

On the other it did not launch the app or run the tests but showed as everything good.

 Task :app:connectedDebugAndroidTest
        : 
        : BUILD SUCCESSFUL in 3s

Note no starting tests steps on the above.

It looks like I eventually fixed this with Flutter clean and reloading dependencies but felt it worth flagging as it could potentially do with some sort of warning or more information on Patrol test if no test are actually initiated.

I noticed someone else has logged the same anomaly and suggested it was fixed by adding MainActivityTest.java and correct directory structure but I had those in place so at this point I feel its more linked to flutter clean and something that corrects.

Would be interested in some thoughts as to why it actually happens?

**Comment by RashmiShashindra** [comment]
Hi @AndyCodemate, I am also having the same problem and tried the flutter clean as well. But still no luck. Getting the below error. Appreciate if you could help.

` > Task :app:connectedStagingDebugAndroidTest
        : Starting 0 tests on Nexus_6_API_30(AVD) - 11
        Instrumentation did not complete:
        Instrumentation code: 0
        Instrumentation result bundle: 
        : 
        : > Task :app:connectedStagingDebugAndroidTest FAILED
`

CC: @bartekpacia

**Comment by bartekpacia** [comment]
Hi, this is most likely related to a mistake during [native setup](https://patrol.leancode.co/getting-started/native#integrate-with-native-side).

See #1631 for a similar problem (possible the same). Make sure your files are in correct directories.

**Reply by RashmiShashindra** [reply]
@bartekpacia Thank you for this. It worked for me. 👍
